packages/react-layout-builder/src/index.js

```diff
@@ -1,16 +1,14 @@
 import React from 'react';
 
-var layoutCount = 0;
 export const layout = (...children) => {
   return (
-    <div key={`layout-${layoutCount++}`} className="layout">
```

Comment:
this causes re-rendering bug, form is re-rendered everytime to a new number when props changes